notebooks/Datasets_1_Prepare_PubChem_Datasets.ipynb

```diff
@@ -99,10 +99,19 @@
    "metadata": {},
```

Comment:
<p>Boom! Amazing. Thanks to @jacob-umans...</p><p>To be explicit, I would probably do <code>if </code>row.acname == 'IC50' and <code>row.aid in IC50_to_Ki </code>just to make sure we don't accidentally convert something that's already in Ki.  Perhaps I'm not understanding and that condition is already implicit, but can't hurt.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/srensi/tmprss2/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
done